kfp/kfp_ray_components/Makefile

```diff
@@ -31,6 +31,10 @@ reconcile-requirements::
 	sed -i.back "s/kfp-data-processing:[0-9].*/kfp-data-processing:${KFP_DOCKER_VERSION}/" cleanupRayComponent.yaml
```

Comment:
did we decide to change sed commands to "%" ?

Reply:
Fixed (changed to /") Thanks